Chia-Network/bls-signatures
# Ms.refactor

- Reenable old c++ tests, remove ones no longer relevant
- Unhardened keys with tests
- Rewrite readme for new API
- Reenable benchmarks
- Rename SignNative to Sign (remove old Sign)
- Rename FromSeed to KeyGen
- Add const wherever possible
- Ensure subgroup membership check is always called
- Other small improvements to API and bug fixes
- Remove obsolete code
